mobile-app/src/firebase/api.ts

```diff
@@ -404,3 +415,60 @@ export async function searchUsersInDb({
     ),
   );
 }
+
+export async function updateProfileDataInDB(
+  profileFormDataAtom: PrimitiveAtom<{
+    displayName: string;
+    username: string;
+  }>,
+  currentUserAtom: PrimitiveAtom<userWithIdT>,
```

Comment:
u dont need to pass this in, you alr have access to this atom (look around in the file)